Documentation/Signoffs/Voice Activity Detection Subsystem.md

```diff
@@ -0,0 +1,108 @@
+
+
+The BOB-19389 datasheet states a signal to noise ratio (SNR) of 67 dBV/Pa taken from a 94 dB SPL signal. In the case of BOB-19389 this is 94 - 67 which equals 27 dB. This means that from the meter distance the microphone will be able to pick up a human whisper of 30 dB without the signal being distored.      
+
+
+(3)
+
+The datasheet of the BOB-19389 also states a 3dB rolloff frequency range of 7 Hz - 19 kHz which includes the threshold set by the constraints to capture human voice frequencies. The BOB-19389 will be able to capture the 100-3000 Hz range set by the constraint.
+
+(4),(5)
+
+To see the sound level output of the speaker, the intensity of the speaker must first be calculated. The equation for intensity is I = P / (4 * pi * r) where P is the power of the system and r is the distance from source. The COM-18343 datasheet states the speaker includes 2 X 2 W speakers and the constarint puts the distance at 1 meter. The intensity can then be calculated to be 0.3183 W/m^2. This value can be converted to dB scle using the equation dB = 10 * log (I / Io), where I is intensity calcuted and Io is the threshold intensity for human hearing which is set to 1*10^-12 W/m^2. The total sound level capable of being outpued by the speaker is 115 dB. With the drone noise being around the value of 96 dB, this means the COM-18343 is able to output sound that is louder than the drone noises this system will be attached to.    
+The COM-18343 datasheet also states the speaker is full range and captures all the heard frequencies from 20 Hz - 20kHz set by the constraint above. But more importantly it will be able to produce sound between 2000 Hz through 5000 Hz which is the range most sensitive to the human ear. 
+
+(6)
+
+The COM-18343 speakers will be able to overcome the noises outside this system such as drone noise, but what about the BOB-19389 microphone? The BOB-19389 is an omnidirectional microphne which is useful to capture sounds that are not necessarily under the microphone, but will also capture other noises such as the drone it is attached to. To solve this issue the casing for this subsystem can incorperate a soundproof barrier for the back of the micrphone eliminating half of the direction sound can be picked up.
+
+A material such as cork has a coefficent of 0.7 sound abosorption. This means that 70% of sound will be abosrobed while the other 30% will be reflected. With commercial drone noise levels reaching 85 dB only around 25.5 dB of sound will be reflected out. This level is within the noise floor level as described in contraint (2) so the drone noise will seem like normal outside noise to microphone alowing it to pick even whispers. The BOB-19389 datasheet also states a max of 134 dB SPL meaning that even if the full sounds are picked up, it will not damage the microphone.
```

Comment:
Reflection is not the relevant problem. Further, you can't just assume that a cork system exists. You have to design and analyze the cork barrier.